tasks/babel.js

```diff
@@ -11,6 +12,9 @@ module.exports = function (grunt) {
 			delete options.filenameRelative;
 
 			options.sourceFileName = path.relative(path.dirname(el.dest), el.src[0]);
+			if (os.platform() === 'win32') {
```

Comment:
I always felt the os approach is cleaner but since it requires a new
require I can use process platform if you prefer that.
On Sep 20, 2015 00:41, "Sindre Sorhus" <notifications@github.com> wrote:

> In tasks/babel.js
> <https://github.com/babel/grunt-babel/pull/45#discussion_r39923979>:
>
> > @@ -11,6 +12,9 @@ module.exports = function (grunt) {
> >  			delete options.filenameRelative;
> >
> >  			options.sourceFileName = path.relative(path.dirname(el.dest), el.src[0]);
> > +			if (os.platform() === 'win32') {
>
> if (process.platform === 'win32') {
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/babel/grunt-babel/pull/45/files#r39923979>.
>